c2corg_api/views/user.py

```diff
@@ -166,11 +193,13 @@ def __init__(self, request):
             partial(validate_unique_attribute,
                     "forum_username",
                     lowercase=True),
+            validate_username,
             validate_forum_username,
             validate_captcha])
     def post(self):
         user = schema_create_user.objectify(self.request.validated)
         user.password = self.request.validated['password']
+        user.email = self.request.validated['email']
```

Comment:
I don't think this line is useful, as you make no change on the email attribute anyway.
And it's whitelisted in https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/user.py#L180
Not sure exactly what it means but I think it means the attribute is directly created from ``self.request.validated`` by the 2 lines above:
``user = schema_create_user.objectify(self.request.validated)``